cfgov/apache/conf.d/redirects.conf

```diff
@@ -1017,8 +964,34 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
+Redirect permanent /how-to-prepare-your-finances-for-buying-a-home/ /owning-a-home/process/prepare/
+RedirectMatch permanent (?i)^/how-to-prepare-your-finances-for-buying-a-home/(.*) /owning-a-home/process/prepare/
+Redirect permanent /owning-a-home/help-us-improve/ /owning-a-home/
+Redirect permanent /owning-a-home/loan-options/FHA-loans/ /owning-a-home/loan-options/fha-loans/
+Redirect permanent /policy-compliance/enforcement/petitions/candy-kern-fuller-and-howard-sutter-iii/ /policy-compliance/enforcement/petitions/candy-kern-fuller-and-howard-sutter-III/
+Redirect permanent /policy-compliance/guidance/supervision-examinations/supervisory-highlights-mortgage-servicing-special-edition-issue-11/ /policy-compliance/guidance/supervisory-highlights/
+Redirect permanent /policy-compliance/guidance/supervisory-guidance/bulletin-safe-act-transitional-licensing-mortgage-loan-originators/ /policy-compliance/guidance/loan-originator-rule/
+Redirect permanent /policy-compliance/notice-opportunities-comment/open-notices/consumer-leasing-regulation-m/ /policy-compliance/notice-opportunities-comment/archive-closed/consumer-leasing-regulation-m/
+Redirect permanent /policy-compliance/notice-opportunities-comment/open-notices/notice-proposed-rulemaking-payday-vehicle-title-and-certain-high-cost-installment-loans/ /policy-compliance/notice-opportunities-comment/archive-closed/notice-proposed-rulemaking-payday-vehicle-title-and-certain-high-cost-installment-loans/
+Redirect permanent /policy-compliance/notice-opportunities-comment/open-notices/request-information-regarding-use-alternative-data-and-modeling-techniques-credit-process/ /policy-compliance/notice-opportunities-comment/archive-closed/request-information-regarding-use-alternative-data-and-modeling-techniques-credit-process/
+Redirect permanent /policy-compliance/rulemaking/rules-under-development/amendments-equal-credit-opportunity-act-regulation-b-ethnicity-and-race-information-collection/ /policy-compliance/notice-opportunities-comment/archive-closed/amendments-equal-credit-opportunity-act-regulation-b-ethnicity-and-race-information-collection/
+Redirect permanent /practitioner-resources/students/knowbeforeyouowe/about/ /practitioner-resources/students/knowbeforeyouowe/
+Redirect permanent /practitioner-resources/students/repay/ /paying-for-college/repay-student-debt/
+Redirect permanent /practitioner-resources/students/request-for-information-on-student-loan-servicing/ /paying-for-college/choose-a-student-loan/
+Redirect permanent /pressrelease/treasury-department-announces-senior-leadership-hires-for-the-consumer-financial-protection-bureau/ /about-us/newsroom/treasury-department-announces-senior-leadership-hires-for-the-consumer-financial-protection-bureau/
+Redirect permanent /regulations/final-remittance-rule-amendment-regulation-e/ /policy-compliance/rulemaking/regulations/1005/
+RedirectMatch permanent (?i)^/supervision/(.*) /jobmanager/supervision.html
```

Comment:
There are a few weird things going on with supervision URLs, though none have to do with the job listing page.

I've added the `/supervision/manual/` ➡️ `/policy-compliance/guidance/supervision-examinations/` redirect in Wagtail.

There are other redirects being triggered by [this rule on L558](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/apache/conf.d/redirects.conf#L558) that are problematic, as well – mod_alias's `Redirect` automatically appends anything after the end of the matched path to the end of the destination path, causing 404ing redirects like:

`/guidance/supervision/manual/ecoa-narrative/` ➡️ `/policy-compliance/guidance/supervision-examinations/ecoa-narrative/`

Please adjust L558 as follows:

```diff
-Redirect permanent /guidance/supervision/manual/  /policy-compliance/guidance/supervision-examinations/
+RedirectMatch permanent ^/guidance/supervision/manual/$  /policy-compliance/guidance/supervision-examinations/
```

And the redirect being added on this line should just be deleted.

```suggestion
```